src/lib/validators.ts

```diff
@@ -26,3 +26,28 @@ export const updateUserSchema = z.object({
   lastName: z.string().min(1),
   member: z.boolean(),
 });
+
+export const insertGameSessionScheduleSchema = createInsertSchema(
+  gameSessionSchedules,
+  {
+    semesterId: z.number(),
+    weekday: z.enum(weekdayEnum.enumValues),
+    startTime: z.string().time(),
+    endTime: z.string().time(),
+    locationName: z.string(),
+    locationAddress: z.string(),
+    capacity: z.number(),
+    casualCapacity: z.number(),
+  },
```

Comment:
No need for overrides -- the create insert schema automatically populates validations for alot of these fields. I think the only thing that might be worth keeping in is z.string().time(), since I'm not sure if drizzle-zod is updates to handle times to that specificity

Reply:
Yep, done!